ui/raidboss/data/06-ew/raid/p7n.txt

```diff
@@ -9,7 +9,7 @@ hideall "--sync--"
 # -ii 77F6 77F7 77FA 77FD 77FF 7800 7804 7806 7807 780A 783E 7841
 
 0.0 "--sync--" sync /Engage!/ window 0,1
-3.0 "--sync--" sync / 1[56]:[^:]*:Agdistis:78E3:/
+3.0 "--sync--" sync / 1[56]:[^:]*:Agdistis:78E3:/ window 3,5
```

Comment:
Autos have inconsistent timing.  They are ok to start a timeline, but you should add windows to them, because it'll catch the next auto.  This timeline already has a `window 16,5` below.  If you wanted to make it earlier, you could add a starts using line.

Reply:
Ok, tightened down the windows on those.